src/reducers/systemIntakesReducer.ts

```diff
@@ -1,19 +1,45 @@
-import { STORE_SYSTEM_INTAKES } from '../constants/actions';
-import { SystemIntakesState } from '../types/systemIntake';
+import { SystemIntakesState } from 'types/systemIntake';
+import { prepareSystemIntakeForApp } from 'data/systemIntake';
+import { fetchSystemIntakes } from 'routines/routines';
+import { DateTime } from 'luxon';
 
 const initialState: SystemIntakesState = {
-  systemIntakes: []
+  systemIntakes: [],
+  loaded: false,
+  loadedTimestamp: null,
+  error: null
 };
 
 function systemIntakesReducer(
   state = initialState,
   action: any
 ): SystemIntakesState {
   switch (action.type) {
-    case STORE_SYSTEM_INTAKES:
+    case fetchSystemIntakes.TRIGGER:
```

Comment:
`fetchSystemIntakes.TRIGGER` seems redundant. It triggers the action, but `.request()` is actually starting the request.

Reply:
I think we can take out `case fetchSystemIntakes.REQUEST`, but I think there's value in having separate actions to initiate the routine (TRIGGER) and make the request (REQUEST). Is that what you're saying?

Reply:
It makes more sense to keep `case fetchSystemIntakes.REQUEST` because it's dispatched in `src/sagas/systemIntakesSaga.ts` right before the API request happens. `REQUEST` is where we would set `isLoading: true`.

There might be a use case for `TRIGGER` in the reducer in the future, but there isn't one now. `TRIGGER` happens when the routine is executed, not necessarily meaning a request is made.

Reply:
Okay, I'll make that change